src/components/NavBar.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const NavBar = () => {
+  return (
+    <div>
```

Comment:
this should be surrounded with `<nav>` to allow people using screen readers to navigate your site

Reply:
Done